schema/deploy/function_get_reporting_year.sql

```diff
@@ -0,0 +1,17 @@
+-- Deploy ggircs-portal:function_get_reporting_year to pg
+-- requires: table_reporting_year
+
+begin;
+
+create or replace function ggircs_portal.get_reporting_year()
+    returns ggircs_portal.reporting_year
+    as $function$
+
+            select *
+            from ggircs_portal.reporting_year
+            where (select CURRENT_TIMESTAMP) between reporting_period_start::timestamp with time zone and reporting_period_end::timestamp with time zone
```

Comment:
This seems incorrect. We want to return the reporting year 2019 if the current timestamp is in 2019, but `reporting_period_start` and `reporting_period_end` are in 2018 for that year